src/invoices/InvoiceForm/InvoiceFormContainer.js

```diff
@@ -117,73 +113,69 @@ function InvoiceEditLayer({
     links: invoiceDocuments.filter(invoiceDocument => invoiceDocument.url),
   };
 
-  const hasLoaded = !id || get(resources, 'invoice.hasLoaded');
-
-  const formNode = hasLoaded
-    ? (
-      <Fragment>
-        <InvoiceForm
-          stripes={stripes}
-          initialValues={initialValues}
-          connectedSource={connectedSource}
-          parentResources={parentResources}
-          parentMutator={parentMutator}
-          onSubmit={saveInvoiceHandler}
-          onCancel={closeScreen}
-        />
-        {
-          isNotUniqueOpen && (
-            <ConfirmationModal
-              id="invoice-is-not-unique-confirmation"
-              heading={<FormattedMessage id="ui-invoice.invoice.isNotUnique.confirmation.heading" />}
-              message={<FormattedMessage id="ui-invoice.invoice.isNotUnique.confirmation.message" />}
-              onCancel={() => {
-                toggleNotUnique();
-                setForceSaveValues(null);
-              }}
-              onConfirm={() => saveInvoiceHandler(forceSaveValues)}
-              open
-            />
-          )
-        }
-      </Fragment>
-
-    )
-    : <LoadingPane onClose={closeScreen} />;
-
-  return isCreate
-    ? formNode
-    : (
-      <Layer
-        contentLabel={intl.formatMessage({ id: 'ui-invoice.invoice.editLayer' })}
-        isOpen
-      >
-        {formNode}
-      </Layer>
+  const hasLoaded = !id || get(resources, 'invoiceFormInvoice.hasLoaded');
+
+  if (!hasLoaded) {
+    return (
+      <Paneset>
+        <LoadingPane onClose={onCancel} />
+      </Paneset>
     );
+  }
+
+  return (
+    <Fragment>
```

Comment:
<>

Reply:
really? :) the rest places are defined with `Fragment`

Reply:
AFAIK the FOLIO pattern is to use `<>`

Reply:
we do have a rule in stripes-acq-components, `"react/jsx-fragments": "off",` - and it was done because it requires a lot of changes to the modules, so it's a tech-debt to remove this rule to be compliant with the rest FOLIO platform code style

Reply:
mb it's better to change rule to have warning? with warning it's much easier to replace fragments during development